src/network/clientpackethandler.cpp

```diff
@@ -895,8 +895,10 @@ void Client::handleCommand_DetachedInventory(NetworkPacket* pkt)
 		inv = inv_it->second;
 	}
 
-	std::string contents;
-	*pkt >> contents;
+	u16 ignore;
```

Comment:
If you do something like this in there, put a "poison" value in there, like 0xdead in this case would work well.

Reply:
this would break compatibility with 5.0.0